Makefile

```diff
@@ -197,6 +202,11 @@ certs/envoycert.pem: certs/CAkey.pem certs/envoykey.pem
 		-days 1825 -sha256 \
 		-extfile _integration/cert-envoy.ext
 
+.PHONY: site-devel
+site-devel: ## Launch the website in a Docker container
+	docker run --publish $(JEKYLL_PORT):$(JEKYLL_PORT) -v $$(pwd)/site:/site -it $(JEKYLL_IMAGE) \
```

Comment:
The 2nd $(JEKYLL_PORT) is probably incorrect here unless we change it in the image.

Reply:
Meaning if I choose port 5000 instead, the container will still listen on 4000

Reply:
Yeah, the excerpt should be `--publish $(JEKYLL_PORT):4000`, that will ensure that port 4000 on the container goes out to whatever you set in $JEKYLL_PORT.

Reply:
Well, you set $JEKYLL_PORT and it works. The fact that it sets the internal and host ports to the same value doesn't matter does it? FWIW, this doesn't depend on the value of the `EXPOSE` in the Docker image spec.